frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containersList.component.tsx

```diff
@@ -167,6 +179,7 @@ export const ContainersList = ({
 												onChange={(event) => {
 													// eslint-disable-next-line no-console
 													console.log('handle favourite click', event.target.value);
```

Comment:
Is this console log still needed?

Reply:
Of course not, thanks for catching that.